### DISCUSSION TITLE
Github student developer pack application problem

### DISCUSSION THREAD

**Discussion by ehtishampbg** [community/community#121909] [GitHub Education]
### Hi everyone,

I am Student at the University of Sargodha. Pakistan but currently I am in Taiwan for my research. As i proceed to apply for studnet developer pack it is being rejected due to the location. What should I do now, I have both IDs i.e., issued by University and by the research station where i am working now. Please help

**Comment by seamoonpandey** [comment]
First you need to add a billing address and then manage the 2fa in you gh account.Then you need to verify your ID card by taking a live photograph of your id card with the device you are filling the form and you will be free to fill in with any emails too. Worked for me.

**Comment by nikeedev** [comment]
> ### Hi everyone,
> I am Student at the University of Sargodha. Pakistan but currently I am in Taiwan for my research. As i proceed to apply for studnet developer pack it is being rejected due to the location. What should I do now, I have both IDs i.e., issued by University and by the research station where i am working now. Please help

Hi @ehtishampbg, I don't know how you can fix the issue, so a better plan is to contact Github on their support page, they can possibly resolve the feature so that you can send in the application even if you are researching outside of the country of the university: https://support.github.com/request/education
